stdlib/prelude.ysh

```diff
@@ -0,0 +1,15 @@
+# These were helpful while implementing args.ysh
+# Maybe we will want to export them in a prelude so that others can use them too?
+#
+# Prior art: Rust has `todo!()` which is quite nice. Other languages allow
+# users to `raise NotImplmentedError()`.
```

Comment:
Let's try these out for awhile

I added some comments in the code about this

```
# - 'pass' can be : or true in shell.  It's a little obscure / confusing, but
#   there is an argument for minimalism.  Although I prefer words like 'true',
#   and that already means something.
# - todo could be more static?  Rust presumably does it at compile time

```

Oh actually I JUST remembered that we used to have a `pass` **keyword**.  It was for left-to-right arg passing.

But then people complained that it broke their scripts -- `pass` already means something, it asks for a password or something

So yeah I'm thinking this will be problematic by default

Damn the flat shell namespace!